code/modules/recycling/conveyor2.dm

```diff
@@ -219,15 +222,9 @@
 
 	spawn(5)		// allow map load
 		conveyors = list()
-		for(var/obj/machinery/conveyor/C in machines) //This works if it's: `C in world` , but that's bad practice. Best to try to fix it.
+		for(var/obj/machinery/conveyor/C in machines)
 			if(C.id == id)
-				conveyors += C  //Moving this to initialise to see if it gets around the bug... do we need the spawn thing? Let's find out.
-
-/*/obj/machinery/conveyor_switch/initialize() //This whole thing is new
-	conveyors = list()
-	for(var/obj/machinery/conveyor/C in world) //This works if it's: `C in world` , but that's bad practice. Best to try to fix it so it's in machines
-		if(C.id == id)
-			conveyors += C*/
+				conveyors += C

```

Comment:
This block can now go in `initialize`, instead of a `spawn(5)` in `New` that'd be vulnerable to tearing if maploaded